src/components/table/column-picker/column-picker.component.ts

```diff
@@ -15,7 +44,7 @@ export class ColumnPickerComponent {
     @Input() locked: ReadonlyArray<string> = [];
 
     /** Define a list of columns that are not selected or locked */
-    @Input() deselected: ReadonlyArray<string> = [];
+    @Input() deselected: ReadonlyArray<ColumnPickerValue> = [];
```

Comment:
I don't think we can change the type of this or the type of `selectedChange`. 
We should probably change this to `string | ColumnPickerValue` which should continue to use strings if people were previously using strings and ColumnPickerValue if it is being used. Changing this to always be `ColumnPickerValue` would be a breaking change.